yaml/README.md

```diff
@@ -0,0 +1,34 @@
+# yaml
+
+A Wing library for working with yaml.
+
+## Prerequisites
+
+* [winglang](https://winglang.io).
+
+## Installation
+
+```sh
+npm i @winglibs/yaml
+```
+
+## Usage
+
+```js
+bring fs;
+bring yaml;
+bring cloud;
+
+let file = fs.readFile("./test.yml");
+let value = yaml.parseValue(file);
+yaml.stringifyValue(json);
+
+new cloud.Function(inflight () => {
+  let value = yaml.parse(file);
+  yaml.stringify(json);
+});
+```
```

Comment:
We should totally include this in the built-in standard library!

Reply:
Don't we already have some yaml stuff in the standard library?

Reply:
I agree this should be part of the stdlib.